vm-virtio/src/vhost_user/blk.rs

```diff
@@ -86,6 +86,11 @@ impl<'a> Blk {
             | 1 << VIRTIO_F_VERSION_1
             | VhostUserVirtioFeatures::PROTOCOL_FEATURES.bits();
 
+        if config_wce == 1 {
```

Comment:
Why `== 1`? Should not you check for `> 0` to cover any other value from `wce`?

Reply:
wce is only 0 or 1, let me change it to bool type, thanks

Reply:
change the wce to bool type now